hr_attendance_management/models/hr_employee.py

```diff
@@ -172,6 +184,18 @@ def _compute_balance(self, store=False):
                                    previous_period_id,
                                    lost,
                                    employee.extra_hours_continuous_cap)
+        return True
+
+    def condition(self):
+        def predicate():
+            return self.periods_computed
+        if not predicate():
+            # _logger.info("Computing periods")
+            self._compute_periods()
```

Comment:
Now the `_compute_periods` is called twice.
`api.depends` → `_compute_periods`
`api.depends` → `_compute_balance` → `condition` → `_compute_periods`